app/helpers/blazer/base_helper.rb

```diff
@@ -39,5 +45,23 @@ def blazer_js_var(name, value)
     def blazer_series_name(k)
       k.nil? ? "null" : k.to_s
     end
+
+    private
+
+    def format_json_value(value)
+      begin
```

Comment:
The use of begin...rescue inside methods is redundant in Ruby since you can directly use rescue

```
def format_json_value(value)
  JSON.pretty_generate(JSON.parse(value))
rescue JSON::ParserError
  value
end

def format_yaml_value(value)
  YAML.dump(YAML.load(value))
rescue StandardError
  value
end
```

Reply:
Good point, done